pyproject.toml

```diff
@@ -1,7 +1,15 @@
 [tool.ruff]
 line-length = 100
 target-version = "py38"
-select = ["I"]
+select = ["W", "E", "F", "I"]
+ignore = [
+   "E711", # comparison to None should be 'if cond is None:'
```

Comment:
Turns out a lot of former disabled checks are not in ruff or do not need to be ignored.